src/components/Table/GridTable.stories.tsx

```diff
@@ -1162,6 +1162,98 @@ export function SelectableRows() {
   );
 }
 
+export function SelectableChildrenRows() {
+  type HeaderRow = { kind: "header"; id: string; data: undefined };
+  type ParentRow = { kind: "parent"; id: string; data: { name: string } };
+  type ChildRow = { kind: "child"; id: string; data: { name: string } };
+  type GrandChildRow = { kind: "grandChild"; id: string; data: { name: string } };
```

Comment:
Can you remove the HeaderRow? Both the test and story are super-long for a 2-line code change, so just looking for ways to make this more succinct.